jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/java/org/jbpm/casemgmt/impl/command/CaseCommand.java

```diff
@@ -30,6 +30,7 @@
 public abstract class CaseCommand<T> implements ExecutableCommand<T> {
 
     private static final long serialVersionUID = 4116744986913465571L;
+    protected static final String ENTRY_POINT_VAR_NAME = "_EntryPoint_";
```

Comment:
Have some doubts about using a variable for storing this, as this will result in an internal implementation detail be very visible in the process instance details.